ddtrace/internal/ci_visibility/_api_client.py

```diff
@@ -43,13 +43,16 @@
 from ddtrace.internal.test_visibility._internal_item_ids import InternalTestId
 from ddtrace.internal.test_visibility.coverage_lines import CoverageLines
 from ddtrace.internal.utils.formats import asbool
-from ddtrace.internal.utils.http import ConnectionType
 from ddtrace.internal.utils.http import Response
 from ddtrace.internal.utils.http import get_connection
 from ddtrace.internal.utils.http import verify_url
 from ddtrace.internal.utils.time import StopWatch
 
 
+if t.TYPE_CHECKING:
+    from ddtrace.internal.utils.http import ConnectionType
```

Comment:
do references to this type need to be changed to `"ConnectionType"` since it isn't defined at runtime?

e.g.

```
conn: t.Optional[ConnectionType] = None
```

->

```
conn: t.Optional["ConnectionType"] = None
```